homeassistant/components/neato/camera.py

```diff
@@ -43,7 +43,6 @@ class NeatoCleaningMap(Camera):
 
     def __init__(self, hass, robot):
         """Initialize Neato cleaning map."""
-        super().__init__()
```

Comment:
We still need this here. The camera base entity is an exception.

Reply:
Oh okay. And how about the switch?

Reply:
There's no init method in the switch base entity class.